### PR TITLE
[Reviewer: Alex] Centos doc changes

### PR DESCRIPTION
**Pull request by eleanor-merry**
These changes document how to build the Chronos package (note that things like the zmq install should really change, but I wanted to get the current process written up)

### CODE REVIEW

File: doc/Development.md [hunk 1]
```diff
@@ -12,6 +12,22 @@ required dependencies are installed.
...
+
+2.  Build and install the libzmq package
+
+        git clone --recursive git@github.com:zeromq/libzmq.git
+        cd libzmq
+        ./autogen.sh
+        ./configure
+        make
+        sudo make install
+        cp /usr/local/lib/libzmq.so.5 /var/lib/jenkins/workspace/chronos-centos/usr/lib
```
Comment (AlexHockey) [inline doc/Development.md]: Just to check - we're not going to release this doc with the jenkins path in it?

Comment (AlexHockey) [inline doc/Development.md]: Does this capture all the symlinks we might have to the zmq library? Can you pass a flag to `make install` telling it to install to the chronos directory? Would that be better?

**Comment by AlexHockey** [comment]
Development.md has some text about "this is only tested on ubuntu" (or words to that effect) which should be updated as well. I have a couple of comments inline, but otherwise looks good.

File: doc/Development.md [hunk 2]
```diff
@@ -12,6 +12,22 @@ required dependencies are installed.
 Chronos depend on a number of tools and libraries.  Some of these are
 included as git submodules, but the rest must be installed separately.
 
+On CentOS 7,
+
+1.  Install the required packages
+
+        sudo yum install libtool gcc-c++ cmake libevent-devel zlib-devel boost-devel net-snmp-devel valgrind python-flask python-requests
```
Comment (rkd-msw) [inline doc/Development.md]: I needed 'git' in this list too (and we specify it for Ubuntu).

Comment (rkd-msw) [inline doc/Development.md]: Also `rpm-build`.

**Comment by eleanor-merry** [comment]
I've not fixed up the ZMQ stuff you mentioned in this PR. There's more work needed to actually make this build on CentOS (e.g. adding libzmq as a submodule, adding make rpm/make rpm move targets, fixing up shared object paths).

**Pull request merged**